docs/migration-guides/external-providers-primitives-v2.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: Migrate provider interfaces from backend.run to primitives
+description: Migrate to the primitives interface in external providers
+
+---
+
+# Migrate third-party provider interfaces from `backend.run` to primitives
+
+## Why should third-party providers implement primitives?
+
+The Qiskit SDK is built to support third parties in creating external providers of quantum resources. 
+The evolution of [user access patterns for QPUs](/migration-guides/qiskit-runtime#why-migrate-to-qiskit-runtime-primitives) 
+has led to the Qiskit primitives (`Sampler` and `Estimator`) superseding the `backend.run` interface.  
+To ensure users of Qiskit can seamlessly transition between providers, we encourage all third-party providers 
+to implement native support of the primitives for performant access to their hardware or simulators. 
+
+This guide is for you if the following applies:
+
+- You are in charge of a third-party provider
+- Your interface is still based on `backend.run`
+- You are looking for guidance to implement your own native `Sampler` and `Estimator` primitives
+
+Once you have migrated your interface to primitives, your users with primitives-based code can seamlessly transition between providers.
+
+
+## Exposing native `Sampler` and `Estimator` implementations
+
+Qiskit defines base interfaces ([BaseEstimatorV2](/api/qiskit/qiskit.primitives.BaseEstimatorV2)
+and [BaseSamplerV2](/api/qiskit/qiskit.primitives.BaseSamplerV2)) that set the standards for input and output handling 
+in the primitives. Native primitive implementations can be developed to wrap any service provider hardware access function
+(for example, `execute_workload(QPU)` or `resource.access()`) or local simulator,
+as long as the final inputs and outputs conform to the established standards set by the base interfaces.
+
+The `Estimator` interface exposes a tool to calculate expectation 
+values of observables with respect to quantum states, while the `Sampler` outputs the counts or probability distributions 
+obtained from sampling from a quantum state in a QPU. These base interfaces also provide standard methods for execution, 
```

Comment:
"obtained from sampling from a quantum state in a QPU" is a bit awkward - does it change the meaning to say instead, "obtained by sampling a quantum state in a QPU" or something like this? @ElePT

Reply:
absolutely, thanks!